index.html

```diff
@@ -19,7 +24,12 @@
         <h2>Welcome</h2>
       </a>
       <nav class="dropbox">
-        <a href="#"><img id="drp-img" src="./images/dropbox.png" alt="dropbox"></a>
+        <a href="#"><img id="drop-image" src="./images/dropbox.png" alt="dropbox"></a>
+        <div class="desktop-toolbar">
+          <a href="#"><h2 id="port-toolbar">Portfolio</h2></a>
+          <a href="#"><h2 id="abou-toolbar">About</h2></a>
+          <a href="#"><h2 id="cont-toolbar">Contact</h2></a>
+        </div>  
```

Comment:
- [ ] Hello, you have done a great job working on this project 💪🏻! I think it would be better if you made this a `ul` element with `li` tags containing the links. This would encourage the use of more semantic HTML tags the overall readability of your code.